pkgs/by-name/ue/uesave/package.nix

```
@@ -0,0 +1,24 @@
+{
+  fetchFromGitHub,
+  lib,
+  rustPlatform,
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "uesave";
+  version = "0.5.0";
+  src = fetchFromGitHub {
+    owner = "trumank";
+    repo = "uesave-rs";
+    rev = "v${version}";
+    sha256 = "sha256-9gOOSLejVfR1KJMhcNuKDkuTOvPC6sNG8xQOZlt8NxI=";
+  };
+
+  cargoHash = "sha256-U6RzSS2j6FK70OHlmWmHZZYT3UB0+Hi+uLofLy+XtGQ=";
+
+  meta = {
+    maintainers = with lib.maintainers; [ xddxdd ];
+    description = "Library for reading and writing Unreal Engine save files (commonly referred to as GVAS).";
```

```suggestion
    description = "Reading and writing Unreal Engine save files (commonly referred to as GVAS)";
```

While itself is a library, this package isn't used as a library.